File: net472/webapi/BuildableWebApi/BuildableWebApi/Controllers/DataReadController.cs
using System.Web.Http;
using System.Web.Http.OData;

namespace BuildableWebApi.Controllers
{
    using System.Web.Http;
    using System.Web.Http.OData;

    public class DataReadController : ApiController
    {
        [HttpGet]
        [EnableQuery(PageSize = 10)]
        [Route("")]
        public IHttpActionResult Get()
        {
            return (IHttpActionResult)Ok();
        }
    }
}

File: net472/VBBuildableWebApi/VBWebApi/Controllers/DataReadController.vb
Imports System.Web.Http
Imports System.Web.Http.Odata

Namespace Controllers
    Public Class DataReadController
        Inherits ApiController
    
        <HttpGet>
        <Route("")>
        Public Function [Get]() As IHttpActionResult

            Return TryCast(Ok(), IHttpActionResult)

        End Function
    End Class


End NameSpace